package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "openfisca.org",
+  "version": "1.0.0",
+  "description": "The public-facing website of OpenFisca for an international audience.",
+  "main": "publish.js",
+  "dependencies": {
+    "gh-pages": "^1.0.0"
+  },
+  "devDependencies": {
+    "gh-pages": "^1.0.0"
```

Comment:
This is redundant with the `dependencies` above. This is most probably only used as a `devDependencies`.